src/main/docker/Dockerfile.postgresql

```diff
@@ -0,0 +1,5 @@
+FROM postgres:14.1
```

Comment:
Without using custom images, I don't think it can't be done as for the wildfly/eap images. I created the 2 files to avoid duplicating the whole <image> tag.

I think I should do this to achieve this without the separated Dockerfiles (notice also the different variables inside the `skip` tags, otherwise 2 containers would be started):

```
<image>
    <name>entando/${project.artifactId}-${db.vendor}:${project.version}</name>
    <alias>${project.artifactId}-dbms</alias>
    <build>
        <from>mysql:8.0.28</from>
        <skip>${skipMySQLDatabaseImage}</skip>
        <env>
            <MYSQL_DATABASE>entando</MYSQL_DATABASE>
            <MYSQL_USER>username</MYSQL_USER>
            <MYSQL_PASSWORD>password</MYSQL_PASSWORD>
            <MYSQL_ALLOW_EMPTY_PASSWORD>true</MYSQL_ALLOW_EMPTY_PASSWORD>
        </env>
    </build>
    <run>
        <skip>${skipMySQLDatabaseImage}</skip>
        <namingStrategy>alias</namingStrategy>
        <network>
            <mode>custom</mode>
            <name>${project.artifactId}-network</name>
            <alias>${project.artifactId}-dbms</alias>
        </network>
        <ports>
            <port>+docker.host.address:db.server.local.port:${db.server.port}</port>
        </ports>
        <wait>
            <tcp>
                <ports>
                    <port>${db.server.port}</port>
                </ports>
            </tcp>
            <time>120000</time>
        </wait>

        <log>
            <enabled>true</enabled>
            <prefix>dbms:</prefix>
            <color>cyan</color>
        </log>
    </run>
</image>
<image>
    <name>entando/${project.artifactId}-${db.vendor}:${project.version}</name>
    <alias>${project.artifactId}-dbms</alias>
    <build>
        <from>postgres:14.1</from>
        <skip>${skipPostgreSQLDatabaseImage}</skip>
        <env>
            <POSTGRES_DB>entando</POSTGRES_DB>
            <POSTGRES_USER>username</POSTGRES_USER>
            <POSTGRES_PASSWORD>password</POSTGRES_PASSWORD>
        </env>
    </build>
    <run>
        <skip>${skipPostgreSQLDatabaseImage}</skip>
        <namingStrategy>alias</namingStrategy>
        <network>
            <mode>custom</mode>
            <name>${project.artifactId}-network</name>
            <alias>${project.artifactId}-dbms</alias>
        </network>
        <ports>
            <port>+docker.host.address:db.server.local.port:${db.server.port}</port>
        </ports>
        <wait>
            <tcp>
                <ports>
                    <port>${db.server.port}</port>
                </ports>
            </tcp>
            <time>120000</time>
        </wait>

        <log>
            <enabled>true</enabled>
            <prefix>dbms:</prefix>
            <color>cyan</color>
        </log>
    </run>
</image>
```

Am I missing something?

Reply:
OK agree. Let's just use resource filtering to interpolate variables in Dockerfiles